lib/vauth/digest.c

```diff
@@ -352,7 +352,7 @@ CURLcode Curl_auth_create_digest_md5_message(struct Curl_easy *data,
   char algorithm[64];
   char qop_options[64];
   int qop_values;
-  char cnonce[33];
+  char cnonce[17];
```

Comment:
This function is used by curl_sasl.c, so not related to http, therefore better leave this cnonce as it was.